### PR TITLE
ADD mne-python

### PR DESCRIPTION
**Pull request by teonbrooks**
This adds mne-python to the flyer.

### CODE REVIEW

File: flier/eppy-handout.tex [hunk 1]
```diff
@@ -200,6 +200,34 @@
...
+%\includegraphics[width=0.3\columnwidth]{../pics/psychopy_logo.pdf}
+%\end{figure}
+
+MNE is a software package for processing magnetoencephalography (MEG) and electroencephalography (EEG) data.
+(\url{http://www.martinos.org/mne})
+\begin{itemize}[nolistsep,topsep=0em,leftmargin=1pc]
+\item Includes MEG and EEG preprocessing tools
+\item Includes various data conditioning and data conversion utilities.
+\item Interactive and batch-mode modules for the forward and inverse calculations
+\item Computes cortically-constrained L2 minimum-norm current estimates
```
Comment (agramfort) [inline flier/eppy-handout.tex]: `Computes cortically-constrained source estimates`

as there is much more than L2 now.

File: flier/eppy-handout.tex [hunk 2]
```diff
@@ -200,6 +200,34 @@
...
+\item Computes associated dynamic statistical parametric maps, optionally constrained by fMRI
+\item Includes frequency-domain and time-frequency analyses and non-parametric statistics
```
Comment (agramfort) [inline flier/eppy-handout.tex:222]: I would add it offers multi-variate statistics (decoding)

**Comment by agramfort** [comment]
besides LGTM

thanks @t3on !

**Comment by agramfort** [comment]
if @yarikoptic is happy I am happy. Thanks @t3on !

**Comment by yarikoptic** [comment]
cool -- great to see MNE-python blazing full speed ahead!
I have fastforwarded locally, shrunk it a bit (as you might have guessed 2 citations would be a bit too much given the space -- chosen neuroimage one for now and shrunk list of authors... sorry), tripped other entries as well but still would need to find some artistic moment to trim even more -- we need to fit into 2 pages for now.  So if anyone is bored -- I would only welcome more of PRs ;)

**Comment by agramfort** [comment]
thanks heaps !

**Comment by yarikoptic** [comment]
now that I have slept a bit -- is there a chance you guys could make a figure with some white/off-white background?  I am often printing those tri-folds locally on our own printer, thus less ink I spend the better ;)

**Comment by agramfort** [comment]
should be doable... maybe just swapping black and white and putting back
some black where needed... but I really suck with gimp :(

**Pull request merged**